src/main.js

```diff
@@ -36,6 +32,19 @@ function randomize() {
     };
   coverImage.src = getRandomIndex(covers);
 }
+
+function showForm() {
+  homeView.classList.add('hidden');
+  formView.classList.remove('hidden');
```

Comment:
This function looks great, we are going to have to add to it in order to hide and show the correct buttons at the top of the page, per the instructions. 

- When the Form view is visible, the “Show New Random Cover” and “Save Cover” buttons should be hidden

- When the Form view is visible, the “Home” button should be visible